source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -68,24 +68,33 @@ To remove a library dependency from a project, select :guilabel:`Manage Current
 Libraries
 ---------
 
-.. warning:: These are **not** links to directly plug in to the :guilabel:`VS Code` -> :guilabel:`Install New Libraries (online)` feature. Click these links to visit the vendor site to see whether they offer online installers, offline installers, or both.
+.. warning:: Many vendors, such as CTRE & REV, recommend using their offline installers over using the online vendor json URL.  This recommendation is because the offline installer is bundled with additional programs that are extremely useful when working with their devices.
+
+Click these links to visit the vendor site to see whether they offer online installers, offline installers, or both.  URLs below are to plug in to the :guilabel:`VS Code` -> :guilabel:`Install New Libraries (online)` feature.  The vendor json will break if a team has not ran an "online" build after 30 days, since the cache will clear.
```

Comment:
"The vendor json will break if a team has not ran an "online" build after 30 days, since the cache will clear."

It would be good to state that this is not the case if the vendor's offline installer is used.

Reply:
Yes that should work.  It's technically a bit more subtle than that (if they install online, and later install the offline...) but that's likely to be more confusing than useful.